models/intermediate/int_recharge__calendar_spine.sql

```diff
@@ -1,20 +1,20 @@
 with spine as (
-    
+
     {# Calculates first and last dates if at least one is not manually set #}
     {% if execute %}
         {% if not var('recharge_first_date', None) or not var('recharge_last_date', None) %}
             {% set date_query %}
-                select  
+                select
                     cast(min(created_at) as {{ dbt.type_timestamp() }}) as min_date,
-                    cast(max(created_at) as {{ dbt.type_timestamp() }}) as max_date 
+                    cast(max(created_at) as {{ dbt.type_timestamp() }}) as max_date
                 from {{ source('recharge','charge') }}
                 {% endset %}
             {% set calc_first_date = run_query(date_query).columns[0][0]|string %}
             {% set calc_last_date = run_query(date_query).columns[1][0]|string %}
         {% endif %}
 
     {# If only compiling, creates range going back 1 year #}
-    {% else %} 
+    {% else %}
         {% set calc_first_date = dbt.dateadd("year", "-1", "current_date") %}
         {% set calc_last_date = dbt.current_timestamp_backcompat() %}
```

Comment:
This just caught my eye. We don't want to use the backcompat version any more. Thank you for helping to modernize Recharge.
```suggestion
        {% set calc_last_date = dbt.current_timestamp() %}
```